### PR TITLE
20929 Remove unnecessary "self debug:#testSelector" comment from test methods - PASS 1

### PR DESCRIPTION
**Pull request by astares**
https://pharo.fogbugz.com/f/cases/20929/Remove-unnecessary-self-debug-testSelector-comment-from-test-methods-PASS-1

### CODE REVIEW

File: src/Collections-Tests/ByteSymbolTest.class.st [hunk 1]
```diff
@@ -9,76 +9,63 @@ Class {
...
-self assert:
-	(Symbol allSymbols select: [:e | e asString  = dt  ]  ) size = 1 .
-self assert:
-	(ByteSymbol allInstances select: [:e | e = dt ] ) size = 1 .
-	
+	| dt newFrom1 newFrom2 |
+	dt := DateAndTime now asString.
+	newFrom1 := ByteSymbol newFrom: dt.
+	self assert: (Symbol allSymbols select: [ :e | e asString = dt ]) size = 1.
+	self assert: (ByteSymbol allInstances select: [ :e | e = dt ]) size = 1.
```
Comment (pavel-krivanek) [inline src/Collections-Tests/ByteSymbolTest.class.st]: equals: would be nice but let's do it with some next PR

Comment (astares) [reply src/Collections-Tests/ByteSymbolTest.class.st]: I do them separately in an own pass

File: src/Collections-Tests/TDictionaryIncludesWithIdentityCheckTest.trait.st [hunk 2]
```diff
@@ -20,12 +20,6 @@ TDictionaryIncludesWithIdentityCheckTest >> nonEmpty [
 ^self explicitRequirement
 ]
 
-{ #category : #requirements }
-TDictionaryIncludesWithIdentityCheckTest >> nonEmptyWithCopyNonIdentical. [
```
Comment (pavel-krivanek) [inline src/Collections-Tests/TDictionaryIncludesWithIdentityCheckTest.trait.st]: why this method was removed?

File: src/Collections-Tests/DictionaryTest.class.st [hunk 3]
```diff
@@ -248,12 +248,6 @@ DictionaryTest >> nonEmptyDifferentFromNonEmptyDict [
 ^ dictionaryNotIncluded 
 ]
 
-{ #category : #requirements }
-DictionaryTest >> nonEmptyWithCopyNonIdentical. [
```
Comment (pavel-krivanek) [inline src/Collections-Tests/DictionaryTest.class.st]: why this method was removed?

**Comment by pavel-krivanek** [comment]
test failures __are related__

**Comment by astares** [comment]
This is a Bug in Iceberg

**Comment by astares** [comment]
So the problem was solved and the PR only changes what is necessary.  Also the tests pass.

Time for another new review and merge

**Comment by astares** [comment]
Tests fail again - but the PR does not change any code, only comments. I do not understand why the CI is so wacky

**Pull request merged**